src/Microsoft.ML.SearchSpace/Contract.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
We have a `Contracts.cs` file that has an `Assert` method that does what you want already. You would have to make this project dependent on the `Microsoft.ML.Core` project to use it, but since that project is included in the main ML.NET nuget package I think using the contracts we already have is a good idea so we don't duplicate code.

Reply:
KK will do that

Reply:
### Resolved